src/core/models.py

```diff
@@ -245,8 +262,8 @@ def sans(self) -> dict[str, list[str]]:
             return {}
 
         return {
-            "sans_ip": [self.private_ip],
-            "sans_dns": [self.hostname, self.fqdn],
+            "sans_ip": [ip for ip in [self.private_ip] if ip],
+            "sans_dns": [dns for dns in [self.hostname, self.fqdn] if dns],
```

Comment:
```suggestion
            "sans_dns": [dns for dns in {self.hostname, self.fqdn} if dns],
```

Reply:
Interesting, I will commit this change and let's see.